tests/istio.mk

```diff
@@ -70,19 +73,29 @@ DEFAULT_UPGRADE_E2E_ARGS += --target_version=""
 UPGRADE_E2E_ARGS ?= ${DEFAULT_UPGRADE_E2E_ARGS}
 
 # Simple e2e test using fortio, approx 2 min
-e2e_simple: istioctl generate_yaml e2e_simple_run
+e2e_simple: istioctl generate_yaml localregistry_setup e2e_simple_run
+
+e2e_mixer: istioctl generate_yaml localregistry_setup e2e_mixer_run
 
-e2e_mixer: istioctl generate_yaml e2e_mixer_run
+e2e_dashboard: istioctl generate_yaml localregistry_setup e2e_dashboard_run
```

Comment:
not directly related here, but shall we refactor these since all these targets calls "istioctl generate_yaml localregistry_setup". should be able to move them to a common target to reuse?

Reply:
Good point.